test-project-play-rpm/run-test-playbook.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/
+set -euo pipefail
+IFS=$'\n\t'
+
+ansible-playbook --private-key=.vagrant/machines/default/virtualbox/private_key --user=vagrant --connection=ssh --limit='default' --inventory-file=.vagrant/provisioners/ansible/inventory provisioning/test.yml
```

Comment:
See
http://docs.ansible.com/guide_vagrant.html#running-ansible-manually
and
http://docs.vagrantup.com/v2/provisioning/ansible.html